src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -89,6 +89,51 @@ private class ACCT_IndividualAccounts_TEST {
         system.assertEquals(null,insertedContacts[0].AccountId);
     }
     
+    /*********************************************************************************************************
+    * @description Accounts created from the trigger should populate the currency code with the contact 
+    */
+    @IsTest
+    private static void accountCreatedWithCorrectCurrencyCodes() {
+        //Skip the test if Advancement is installed
+        if (ADV_PackageInfo_SVC.useAdv()) return;
+
+        Boolean isMultiCurrencyEnabled = UserInfo.isMultiCurrencyOrganization();
+        String defaultCurrency = UserInfo.getDefaultCurrency();
+
+        Contact newContact = new Contact(
+            lastname = 'CurrencyTest',
```

Comment:
Let's use CamelCase for fields API names, ie `LastName` and `FirstName`.

Reply:
@RadSFDeveloper Thanks.  I've updated the names